build/ci/install-agent-spawn-host.sh

```diff
@@ -33,6 +33,35 @@ while getopts 'i:h:g:u:a:b:' opt; do
   esac
 done
 
+
+flags=()
+# get the information that you need from the toml file if exists
+if test -f "${XDG_CONFIG_HOME}/mongocli.toml"; then
+  IFS="="
+  while read -r name value
+  do
+    name=${name//[[:blank:]]/}
+    if [[ "${name}" == "project_id" ]]; then
+      LC_GROUP_ID=$(echo "${value}" | tr -d '"')
+      export LC_GROUP_ID
+    fi
+
+    if [[ "${name}" == "agent_api_key" ]]; then
+      LC_AGENT_KEY=$(echo "${value}" | tr -d '"')
+      export LC_AGENT_KEY
+    fi
+
+    if [[ "${name}" == "ops_manager_url" ]]; then
+      BASE_URL=$(echo "${value}" | tr -d '"')
+      export BASE_URL
+    fi
+  done < "${XDG_CONFIG_HOME}/mongocli.toml"
+
+  flags+=("--ops-manager")
+fi
```

Comment:
[nb] fair play on getting this done on bash but this goes again with one of my recommendations before where you may better off creating a bash file of the format
```bash
export MCLI_*=...
```
and then executing that file over generating a toml not going to block this PR on this but something to consider